src/AutoRest.CSharp/Common/Output/Models/Types/TypeProvider.cs

```diff
@@ -18,6 +18,7 @@ protected TypeProvider(BuildContext context)
         {
             Context = context;
             _existingType = new Lazy<INamedTypeSymbol?>(() => Context.SourceInputModel?.FindForType(DefaultNamespace, DefaultName));
+            DefaultAccessibility = "public";
```

Comment:
What's the benefit of this change?

Reply:
none, originally it was needed for something, but now its not ill revert